consumer-backend/productpass/src/main/java/org/eclipse/tractusx/productpass/config/PassportConfig.java

```diff
@@ -0,0 +1,46 @@
+/*********************************************************************************
+ *
+ * Catena-X - Product Passport Consumer Backend
```

Comment:
Not sure if this shouldn't be Tractus-X.

Reply:
We had our license headers reviewed by Angelika already and it matches with our current license file. Maybe we can schedule a meeting to revise that again. Thanks for the comment :)